### ISSUE TITLE
[BUG] Current Status % on `Accessibility Checker` auditor view incorrect

### ISSUE THREAD

**Issue by fbarroso24** [IBMa/equal-access#106]
## Title line template: [Title]: Brief description
[BUG] Current Status % on `Accessibility Checker` auditor view incorrect
-->

## What tool(s) are you using?
Accessibility Checker

- [ ] `accessibility-checker-engine`
- [x ] `accessibility-checker`
- [ ] `karma-accessibility-checker`
- [ ] `rule-server`
- [ ] `boilerplates`
- [ ] `other`

## Detailed description
The % displayed in `Current Status` card on the `Accessibility Checker` auditor view shows 83% even though `0 violations` are found.

Otherwise, if the % also takes into account the `Needs Review` then the `description` underneath `Current Status` should be updated.  Currently it states `Percentage of elements with no detected violations.`

On a separate note, it's an interesting topic whether the `Needs Review` should be counted in that % as they may or may not be violations.  So to lower the score initially prior to them being evaluated may not paint an accurate representation of the current status.  However, I do understand that you don't want folks to gloss over the `Needs Review`.  

> Node version:
> Platform: 
Chrome

## Additional information

- Screenshots or code
![image](https://user-images.githubusercontent.com/8657768/83542155-96e18d00-a4c8-11ea-882d-642b023a2dbb.png)

**Comment by drjoho** [comment]
The actual defect is that the Current status is based on both violations and needs review. So the description under the current status should say: Percentage of elements with not detected violations or needs review. So as the first fix for this issue we will correct the description.

**Comment by cpandhi** [comment]
@drjoho yes, please update the description to
‘Percentage of elements with no detected violations or items to review’

**Comment by drjoho** [comment]
This issue has been fixed and is in the master.
